README.md

```diff
@@ -340,6 +379,47 @@ Route::get('twitter/logout', ['as' => 'twitter.logout', function () {
 }]);
 ```
 
+### Twitter API v2 Examples
+
+Get user tweets:
+```php
+// ...
+
+use Atymic\Twitter\Twitter as TwitterContract;
+use Illuminate\Http\JsonResponse;
+use Twitter;
+
+// ... 
+
+public function userTweets(int $userId): JsonResponse
+{
+	$params = [
+		'place.fields' => 'country,name',
+		'tweet.fields' => 'author_id,geo',
+		'expansions' => 'author_id,in_reply_to_user_id',
+		TwitterContract::KEY_RESPONSE_FORMAT => TwitterContract::RESPONSE_FORMAT_JSON,
```

Comment:
Could we default this to json?

Reply:
Yes, made JSOn the default for all API v2 methods